Network/Public/CMakeLists.txt

```diff
@@ -9,7 +9,7 @@ set(SOURCE_FILES
     Include/Network/Message.hpp
     Include/Network/SerializationHandler.hpp
     Include/Network/YasSerializer.hpp
-    Include/Network/iAPI.hpp)
+    Include/Network/IClientAPI.hpp)
```

Comment:
Decline. We have an API. It didn't divide on client or server

Reply:
This API is implemented for whom/what?

Reply:
Understand what an API is first
https://en.wikipedia.org/wiki/API

Reply:
> Understand what an API is first https://en.wikipedia.org/wiki/API

This API is implemented for whom/what?

Reply:
> Understand what an API is first https://en.wikipedia.org/wiki/API

![image](https://user-images.githubusercontent.com/70116007/214062088-c31d7033-d91a-4d04-aa39-e22b9403270c.png)

Reply:
> Understand what an API is first https://en.wikipedia.org/wiki/API

Am I correct in assuming that the API is made for "custom clients" or some other entity? If it's for a custom client, it makes sense to add "Client" prefix to the beginning of the class name, and if it's for another entity, then another prefix.